src/components/views/location/shareLocation.ts

```diff
@@ -42,7 +42,7 @@ export const shareLocation = (
     if (!uri) return false;
     try {
         const text = textForLocation(uri, ts, null);
-        const threadId = relation?.rel_type === RelationType.Thread ? relation.event_id : null;
+        const threadId = relation?.rel_type === THREAD_RELATION_TYPE.name ? relation.event_id : null;
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?